stl/inc/future

```diff
@@ -1316,17 +1310,14 @@ public:
     using _MyStateManagerType = _State_manager<_Ptype>;
     using _MyStateType        = _Packaged_state<_Ret(_ArgTypes...)>;
 
-    packaged_task() noexcept : _MyPromise(0) {}
+    packaged_task() = default;

```

Comment:
No change requested: Wow, a `0`-as-null-pointer was still lurking in the codebase! Nice catch!